imodels/tree/figs.py

```diff
@@ -392,7 +394,7 @@ def predict_proba(self, X):
         preds = np.zeros(X.shape[0])
```

Comment:
Thanks, looks nice to me!

Could you possibly add a small flag to the `predict_proba` function in case someone wants to use the old behavior? Something like `predict_proba(self, X, use_clipped_prediction=False)`

Reply:
@csinva Sure! Can we still default to the current `expit` behavior?

Reply:
@csinva done! See https://github.com/csinva/imodels/pull/150/commits/235eb368ebd7441060425dbbd5b92bf7c1416830

Reply:
Let's make a new release for this change, as it will change some model outputs.